components/front_matter/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 serde = {version = "1.0", features = ["derive"] }
-
+time = { version = "0.3", features = ["macros"] }
```

Comment:
I'll check.

Reply:
We can't remove the crate, or the macro feature, because they're used in for example, front_matter's `parse_datetime`:

https://github.com/getzola/zola/pull/1790/files#diff-313e3525c8ae290a23638c1248f2514eada393c295f005ec3c8702c2f7c521a4R73-R79